cloudformation/ebs-substack.cfn.json

```diff
@@ -1141,24 +1141,16 @@
         },
         "Size": {
           "Fn::If": [
-            "Vol1_UseEBSSnapshot",
+            "Vol1_UseVolumeSize",
             {
-              "Ref": "AWS::NoValue"
-            },
-            {
-              "Fn::If": [
-                "Vol1_UseVolumeSize",
+              "Fn::Select": [
+                "0",
                 {
-                  "Fn::Select": [
-                    "0",
-                    {
-                      "Ref": "VolumeSize"
-                    }
-                  ]
-                },
-                20
+                  "Ref": "VolumeSize"
+                }
               ]
-            }
+            },
+            20
```

Comment:
I don't know whether these 20 are needed or not. Since they are here before my change, and there were default value for EBS volumes at that time. I think the CLI shouldn't ever not pass in a value for volume size before my change either. Maybe put it here is just to make the code safer?